admin-portal-frontend/src/app/components/categoryRoutes.tsx

```diff
@@ -0,0 +1,34 @@
+export type Category = {
+  _id: string;
+  title: string;
+  items: [];
+  type: string;
+};
+
+// Gets and returns all categories from API
+export const getAllCategories = async () => {
+  try {
+    const url = "http://localhost:3001/api/categories";
```

Comment:
have the url domain be stored in a .env, just for future production deployment and such